examples/app_commands/.lightning

```diff
@@ -0,0 +1 @@
+name: test_13
```

Comment:
can you name the app something meaningful?

Reply:
Done !